dodiscover/constraint/_classes.py

```diff
@@ -162,10 +158,10 @@ def fit(self, context: Context) -> None:
         self.X_ = self.context_.data
 
         # initialize graph object to apply learning
-        sep_set = self._initialize_sep_sets(self.init_graph_)
+        self.separating_sets_ = self._initialize_sep_sets(self.init_graph_)
 
         # learn skeleton graph and the separating sets per variable
-        graph, sep_set = self.learn_skeleton(self.context_, sep_set)
+        graph, self.separating_sets_ = self.learn_skeleton(self.context_, self.separating_sets_)
```

Comment:
I've been using leading-underscore to indicate private members - is trailing more idiomatic?

Reply:
I use trailing to follow "fitted" attributes pattern that scikit-learn uses. https://scikit-learn.org/dev/developers/develop.html